org.adempiere.base/src/org/adempiere/impexp/GridTabCSVImporter.java

```diff
@@ -1651,6 +1668,95 @@ private int resolveForeign(String foreignTable, String foreignColumn, Object val
 		}
 		return -3;   // no values found, error ForeignNotResolved
 	}
+	
+	/**
+	 * @param foreignTable
+	 * @param foreignColumn
+	 * @param value
+	 * @param field
+	 * @param trx
+	 * @return null for not found
+	 */
+	private String resolveForeignMultipleSelection(String foreignTable, String foreignColumn, Object value, GridField field, Trx trx) {
+		boolean systemAccess = false;
+		if (!"AD_Client".equals(foreignTable)) {
+			MTable ft = MTable.get(Env.getCtx(), foreignTable);
+			String accessLevel = ft.getAccessLevel();
+			if (   MTable.ACCESSLEVEL_All.equals(accessLevel)
+				|| MTable.ACCESSLEVEL_SystemOnly.equals(accessLevel)
+				|| MTable.ACCESSLEVEL_SystemPlusClient.equals(accessLevel)) {
+				systemAccess = true;
+			}
+		}
+		int thisClientId = Env.getAD_Client_ID(Env.getCtx());
+
+		String trxName = (trx!=null?trx.getTrxName():null); 
+
+		StringBuilder postSelect = new StringBuilder(" FROM ")
+			.append(foreignTable).append(" WHERE IsActive='Y' AND AD_Client_ID=? AND ").append(foreignColumn).append(" IN(");
+		
+		if(!(value instanceof String))
+			return null;
+		
+		String[] values = ((String)value).split(",");
+		List<Object> params = new ArrayList<>();
+		params.add(thisClientId);
+		
+		for(int idx = 0; idx<values.length;idx++) {
+			if(idx==values.length-1)
+				postSelect.append("?)");
+			else
+				postSelect.append("?,");
+			params.add(values[idx]);
+		}
+		
+	  if (field != null ) {
+		if (!Util.isEmpty(field.getVO().ValidationCode)) {
+			String dynamicValid = Env.parseContext(Env.getCtx(), field.getWindowNo(), field.getGridTab().getTabNo(), field.getVO().ValidationCode, false);
+			if (Util.isEmpty(dynamicValid)) {
+				return null;// it's parse error but simple consider like ForeignNotResolved
+			}else {
+				postSelect.append(" AND (").append(dynamicValid).append(")");
+			}
+		}
+		int ref = field.getVO().displayType;
+		int refval = field.getVO().AD_Reference_Value_ID;
+		if (refval > 0 && (ref == DisplayType.Table || ref == DisplayType.Search)) {

```

Comment:
this should be just `if (ref == DisplayType.Table || ref == DisplayType.Search)`